### ISSUE TITLE
[Dev] [508] Verify and update forms to render value to screen reader

### ISSUE THREAD

**Issue by tbolt** [CMSgov/eAPD#2676]
As noted in the 508 compliance review:
> Form value (whatever the user is typing) should also be rendered by screen reader. The input text box should have role=value.

More investigation should be done with this issue before moving forward with a solution.

Additional notes:
* We are using the design.cms.gov Text field - https://design.cms.gov/components/text-field/
* Using Safari + Voice Over, the form field values are being rendered by the screen reader

### This task is done when…
This could include:
 - [ ] A clear understanding of the issue is discovered
 - [ ] A new ticket is created with a clear solution to be implemented
